Runtime/AvatarCreator/Scripts/UI/Elements/LogoutElement.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System;
+using ReadyPlayerMe.Core;
+using UnityEngine;
+using UnityEngine.Events;
+
+namespace ReadyPlayerMe.AvatarCreator
+{
+    /// <summary>
+    /// This class provides all the functionality required to create a basic Logout UI element.
+    /// This is used to log out users from RPM accounts, once they have logged in.
+    /// </summary>
+    public class LogoutElement : MonoBehaviour
+    {
+        private const string TAG = nameof(LoginElement);
+        [SerializeField] private UnityEvent OnLogoutSuccess;
```

Comment:
I suggest making these public like we have in the other elements so that developers can access/assign via C# if they want. 
```suggestion
        public UnityEvent OnLogoutSuccess;
```